benchmarks/Main.hs

```diff
@@ -46,7 +46,7 @@ bench count name a f = do
 
   where
     seqN 0 _ _ = ()
-    seqN n f a = f a `seq` seqN (n - 1) f a
+    seqN n f' a' = f' a' `seq` seqN (n - 1) f' a'
```

Comment:
were these shadowing something else?

Reply:
Yes, specifically the arguments f and a being passed to the top function.